airflow/cli/commands/db_command.py

```diff
@@ -39,6 +40,11 @@
 @providers_configuration_loaded
 def initdb(args):
     """Initializes the metadata database."""
+    warnings.warn(
+        "ActionCommand `init` is deprecated.  Use `migrate` instead to migrate the db and/or "
```

Comment:
```suggestion
        "`db init` is deprecated.  Use `db migrate` instead to migrate the db and/or "
```

This is a better message to show to users.